static/app/views/detectors/datasetConfig/errors.tsx

```diff
@@ -78,4 +82,28 @@ export const DetectorErrorsConfig: DetectorDatasetConfig<ErrorsSeriesResponse> =
   fromApiAggregate: aggregate => aggregate,
   toApiAggregate: aggregate => aggregate,
   supportedDetectionTypes: ['static', 'percent', 'dynamic'],
+  toSnubaQueryString: snubaQuery => {
+    if (!snubaQuery) {
+      return '';
+    }
+
+    const defaultsSorted = DEFAULT_EVENT_TYPES.toSorted();
+    const current = snubaQuery.eventTypes;
+    const sameAsDefaults =
+      current.length === defaultsSorted.length &&
+      current.toSorted().every((v, i) => v === defaultsSorted[i]);
```

Comment:
Should we handle the case where it is in a different ordering ([default, error])?